modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -23,11 +23,11 @@ Type::
 
 | `apiVersion`
 | `string`
-| APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and might reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+| APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```

Comment:
Should we use "might" instead of "may" upstream?

Reply:
Yes